### DISCUSSION TITLE
models for specific language?

### DISCUSSION THREAD

**Discussion by alighamdan** [openai/whisper#1800] [Q&A]
when will be model.X like base.en we can find for example base.ar? and large models with specific models?

**Comment by jongwook** [comment]
This is not planned, but you can find many fine-tuned models at: https://huggingface.co/models?search=whisper

**Reply by BraveSpeaker** [reply]
Hello, 
is there a way to FILTER the results? There 6741 models right now, and I don't use HF that much. Are there websites that categorize those whisper models?
